src/firebaseLib.php

```diff
@@ -26,6 +26,7 @@ class FirebaseLib implements FirebaseInterface
     private $_timeout;
     private $_token;
     private $_curlHandler;
+    public  $header;
```

Comment:
Make a getter method for this and set accessibility to `private`. External libraries have no need to write to this field, so we shouldn't allow it.